src/setup_payload/ManualSetupPayloadGenerator.cpp

```diff
@@ -23,6 +23,7 @@
 
```

Comment:
It makes me a bit sad to have to fix `ManualSetupPayloadGenerator.cpp` and `QRCodeSetupPayloadGenerator.cpp` for stack usage. That would make the code a bit less readable and harder to maintain.

For the rest of CHIP that definitively makes sense.
For those files I think the initial intent was that the code that creates the manual/QR code was not designed to be shipped on devices. We mostly have used that as an easy way to create a QRCode on the M5Stack while developing. I wonder if instead of fixing those files, it is time to just get those files out of the devices (not in this PR though).

Thoughts ?

Reply:
I have no objection to leaving it. The main goal of this PR is to make sure it doesn't happen by accident  or get overlooked in new code.

Reply:
So, for this PR I've replaced the ManualSetupPayloadGenerator.cpp with a #pragma and TODO #3663. Then a follow-up PR can remove the TODO and close #3663, so that it's recorded that this stack usage is OK.